.github/workflows/vendor_gardener.yaml

```diff
@@ -0,0 +1,31 @@
+name: Vendor Gardener
+on:
+  push:
+    branches:
+    - dependabot/go_modules/github.com/gardener/**
+  pull_request:
+    branches:
+    - dependabot/go_modules/github.com/gardener/**
+permissions: write-all
+jobs:
+  run:
+    name: Run make revendor
+    runs-on: linux-latest
```

Comment:
Does `linux-latest` exist? I'm not entirely sure, but I can't find it on the [list of available runner-images](https://github.com/actions/runner-images#available-images).

Reply:
my bad at first I used `macos-latest` but after a suggestion of my colleague he told me to use the linux image. `ubuntu-latest` was meant to be used